src/tracer.h

```diff
@@ -20,16 +20,20 @@
 #pragma once
 #include <sys/types.h>
 
-typedef struct files {} files; //TODO
+typedef struct files {
+    size_t size;
+    size_t capacity;
+    char **arr;
+} files; //TODO
```

Comment:
I'm not sure what this is about...

Reply:
This was a plan to create a dynamic container but i postponed it for later, i only forgot removing the structure change but it will be used in my next commit since we concluded that i will handle dynamic memory on my own.